docs/configuration/holmesgpt/toolsets/helm.rst

```diff
@@ -12,6 +12,22 @@ Configuration
         toolsets:
             helm/core:
                 enabled: true
+        customClusterRoleRules:
```

Comment:
Can please add to helm toolsets docs & kubernetes/prom-stack which permissions they should add it for work?